src/client/windshaft.js

```diff
@@ -122,70 +122,76 @@ export default class Windshaft {
         return this._numCategories;
     }
 
-    _instantiate(MNS, resolution, filtering) {
-        let agg = {
-            threshold: 1,
-            resolution: resolution,
+    async _instantiate(minimunRequiredScheme, resolution, filters) {
```

Comment:
The first argument is called in all the project `MNS` or `minimumNeededSchema`, please, don't introduce a new alias.